rust/src/bin/test-drive-net.rs

```diff
@@ -32,7 +25,7 @@ async fn main() -> Result<(), ClientError> {
 
     let mut tasks = vec![];
     for id in 0..10 {
-        let mut client = Client::new_with_addr(1, id, "http://127.0.0.1:3030")?;
+        let mut client = Client::new_with_addr(1, id, "http://127.0.0.1:8081")?;
```

Comment:
we already spoke, just again for reference: would be good to "config" this as well, in a follow-up.

Reply:
Thanks a lot for the review👍 
I created a ticket (PB-837)